### PR TITLE
customized pdf theme for docs module

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

adds a yaml configuration to customize the look and feel of the pdf output. Currently, the only change is in the footers on each (content) page
#### How should this be tested?

review footers, starting at page one of any of the documentation pdf files. It should now include:

Title
Page Number 
Build date

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by shaundmorris** [comment]
@ricklarsen do you happen to have a PDF generated already that we can review this PR?

**Review by rececoffin — APPROVED** [review]

**Comment by ricklarsen** [comment]
> @ricklarsen do you happen to have a PDF generated already that we can review this PR?

this one should be correct: 
[documentation.pdf](https://github.com/codice/alliance/files/7173001/documentation.pdf)

or try this smaller one: 

[introduction.pdf](https://github.com/codice/alliance/files/7173004/introduction.pdf)

**Review by shaundmorris — APPROVED** [review]
Footers look good.

File: distribution/docs/themes/alliance-theme.yml [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    font_color: 181818
+  revision:
+    margin_top: $base_font_size * 1.25
+block:
+  margin_bottom: $vertical_rhythm
+caption:
+  align: left
+  font_size: $base_font_size * 0.95
+  font_style: italic
+  # FIXME perhaps set line_height instead of / in addition to margins?
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/themes/alliance-theme.yml:140]: same as above

File: distribution/docs/themes/alliance-theme.yml [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+  #    size: 24
+  label:
+    text_transform: uppercase
+    font_style: bold
+blockquote:
+  font_size: $base_font_size_large
+  border_color: $base_border_color
+  border_width: 0
+  border_left_width: 5
+  # FIXME disable negative padding bottom once margin collapsing is implemented
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/themes/alliance-theme.yml:174]: I agree. I can take them out.

File: distribution/docs/themes/default-theme.yml [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    text_decoration: underline
+  big:
+    font_size: $base_font_size_large
+  small:
+    font_size: $base_font_size_small
+  subtitle:
+    font_color: 999999
+    font_size: 0.8em
+    font_style: normal_italic
+# FIXME vertical_rhythm is weird; we should think in terms of ems
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/themes/default-theme.yml:68]: 👍

File: distribution/docs/themes/default-theme.yml [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+    font_color: 181818
+  revision:
+    margin_top: $base_font_size * 1.25
+block:
+  margin_bottom: $vertical_rhythm
+caption:
+  align: left
+  font_size: $base_font_size * 0.95
+  font_style: italic
+  # FIXME perhaps set line_height instead of / in addition to margins?
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/themes/default-theme.yml:139]: 👍

File: distribution/docs/themes/default-theme.yml [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+image:
+  align: left
+prose:
+  margin_bottom: $block_margin_bottom
+sidebar:
+  background_color: EEEEEE
+  border_color: E1E1E1
+  border_radius: $base_border_radius
+  border_width: $base_border_width
+  # FIXME reenable padding bottom once margin collapsing is implemented
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/themes/default-theme.yml:224]: yeah, these are the default ones I had to copy in to get my extended one to work.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**